converter/src/test/java/gov/cms/qpp/acceptance/cpc/CpcPlusAcceptanceTest.java

```diff
@@ -60,18 +72,64 @@ void testCpcPlusFileFailures() throws IOException {
 		List<Path> successesThatShouldBeErrors = getXml(FAILURE)
 			.filter(entry -> {
 				Converter converter = new Converter(new PathQrdaSource(entry));
-
+				//TODO remove this once all error scenarios are validated
+				if (fixtureValues.get(entry.toFile().getName()) == null) {
+					return false;
+				}
 				try {
 					converter.transform();
 					return true;
 				} catch (TransformException expected) {
+					//runnning conversions on individual files
+					List<Detail> details = expected.getDetails().getErrors().get(0).getDetails();
+					verifyOutcome(entry.toFile().getName(), details);
 					return false;
 				}
 			}).collect(Collectors.toList());
 
+		assertWithMessage("The fixture file is not representative of the failures directory")
+				.that(getXml(FAILURE).count())
+				.isEqualTo(fixtureValues.size());
 		assertThat(successesThatShouldBeErrors).isEmpty();
 	}
 
+	private void verifyOutcome(String filename, List<Detail> details) {
+		CPCAcceptanceFixture expectedErrors = fixtureValues.get(filename);
+
+		System.out.println("Verifying scenario " + filename +
```

Comment:
Logger please

Reply:
Logger in a test?

Reply:
ah, right... it's a test.  still seems weird to see a raw `System.out`

Reply:
ok

Reply:
Agreed. JUnit 5 has a way to handle this through parameterized tests. You can generate test data in a Stream.